src/systemActs/ContentActs.ts

```diff
@@ -410,3 +411,18 @@ export class InvalidRemoveValueAct<T> extends ContentAct {
         );
     }
 }
+
+export class CanFulfillIntentAct extends ContentAct {
+    payload: CanFulfillIntentPayload;
+
+    constructor(control: Control, payload: CanFulfillIntentPayload) {
+        super(control);
+        this.payload = {
```

Comment:
`this.payload = payload` should work as well
No need to dereference it

Reply:
Will change.